packages/firebase_ml/ios/Classes/FLTModelManager.m

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTFirebaseMLPlugin.h"
+
+@implementation FLTModelManager
+
++ (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
```

Comment:
Is there any support for [deleteDownloadModel](https://firebase.google.com/docs/reference/ios/firebasemlcommon/api/reference/Classes/FIRModelManager#-deletedownloadedmodel:completion:)?

Reply:
As per design doc, deleteDownloadModel had a bug to be fixed before I could implemented. I know that bug was fixed, but I'm not sure if it is in the production yet. 

Either way Android side doesn't have this support at the moment. If we end up implementing deleteDownloadModel for flutter, it will be in a separate PR and will contain IOS, Android and Dart portions altogether.

Reply:
I just confirmed that the deleteDownloadModel native SDK fix push got delayed, and should be available early August.

Reply:
Sounds good, thanks for checking on that.